rules/integrations/aws/collection_s3_unauthenticated_bucket_listing.toml

```diff
@@ -0,0 +1,148 @@
+
+### False Positive Analysis
+
+- **Legitimate Public Access**: Some S3 buckets may intentionally allow public access for specific use cases. Confirm the bucket’s intended configuration with its owner.
+- **Automated Scans or Security Tools**: Public scanners or legitimate tools may access publicly exposed S3 buckets, triggering this rule. Validate the nature of the access.
+
+### Response and Remediation
+
+1. **Immediate Action**:
+   - Restrict or remove public access to the affected S3 bucket.
+     - Update the bucket policy and ensure that access is limited to trusted entities.
+     - Enable AWS S3 Block Public Access settings to prevent further exposure.
+   - Monitor for any additional actions or API calls from the source IP.
+
+2. **Enhance Monitoring**:
+   - Enable detailed logging and monitoring for all S3 buckets to detect similar events in the future.
+   - Configure real-time alerts for unauthenticated or public API calls to sensitive S3 buckets.
+
+3. **Review Security Configuration**:
+   - Audit S3 bucket policies and ACLs across your environment to ensure they follow AWS security best practices.
+   - Remove overly permissive policies and restrict access to trusted principals only.
+
+4. **Analyze for Potential Impact**:
+   - Investigate whether the unauthenticated access attempt was followed by additional suspicious activity.
+   - Review the data stored in the bucket to assess the potential risk of data exposure.
+
+
+### Additional Resources
+
+- [AWS Documentation: S3 Bucket Policy Best Practices](https://docs.aws.amazon.com/AmazonS3/latest/userguide/example-bucket-policies.html)
+- [AWS CloudTrail Best Practices](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-best-practices.html)
+- [AWS S3 Block Public Access](https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html)
+
+"""
+references = [
+    "https://hackingthe.cloud/aws/exploitation/Misconfigured_Resource-Based_Policies/exploting_public_resources_attack_playbook/",
+]
+risk_score = 47
+rule_id = "5d876294-bcb2-11ef-bd29-f661ea17fbce"
+severity = "medium"
+tags = [
+    "Domain: Cloud",
+    "Data Source: AWS",
+    "Data Source: Amazon Web Services",
+    "Data Source: Amazon S3",
+    "Use Case: Asset Visibility",
+    "Resources: Investigation Guide",
+    "Tactic: Collection",
+]
+timestamp_override = "event.ingested"
+type = "new_terms"
+
+query = '''
+event.dataset: "aws.cloudtrail"
+    and event.provider: "s3.amazonaws.com"
+    and event.action: "ListObjects"
+    and event.outcome: "success"
+    and aws.cloudtrail.user_identity.type: ("AWSAccount" or "Unknown")
+    and cloud.account.id: "anonymous"
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1530"
+name = "Data from Cloud Storage"
+reference = "https://attack.mitre.org/techniques/T1530/"
+
+
+[rule.threat.tactic]
+id = "TA0009"
+name = "Collection"
```

Comment:
this looks more of a discovery ctivity vs collection, https://attack.mitre.org/techniques/T1619/